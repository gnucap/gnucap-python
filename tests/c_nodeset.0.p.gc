* Felix Salfelder 2019
* GPLv3+

attach public c_python.so
loadpy c_nodeset.py

options numdgt=10

verilog

parameter v1=3.3
parameter v2=5

nodesetter #(.v_1(v1), .v_2(v2)) ns(1, 2);

print dc v(nodes)
dc
list
nodeset v_1=5 v_2=6
dc
delete ns
list
nodeset
list
