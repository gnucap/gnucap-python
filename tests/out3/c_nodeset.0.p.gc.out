Gnucap : The Gnu Circuit Analysis Package
Never trust any version less than 1.0
Copyright 1982-2013, Albert Davis
Gnucap comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome
to redistribute it under the terms of 
the GNU General Public License, version 3 or later.
See the file "COPYING" for details.
main version
core-lib version
default plugins
generator: already installed, replacing
stashing as generator:0
transient: already installed, replacing
stashing as transient:0
transient: already installed, replacing
stashing as transient:0
#                v(1)            v(2)           
 0.              3.3             5.             
nodesetter #(.v_1(v1),.v_2(v2)) ns (.p0(1),.p1(2));
nodesetter #(.v_1( 5.),.v_2( 6.)) nodesetter (.p0(1),.p1(2));
#                v(1)            v(2)           
 0.              5.              6.             
nodeset
nodesetter #() nodesetter ();
